scripts/qa_crowbarsetup.sh

```diff
@@ -1736,6 +1738,16 @@ function custom_configuration()
                 novanodes="[ ${novanodes%,} ]"
                 proposal_set_value nova default "['deployment']['nova']['elements']['nova-multi-compute-${libvirt_type}']" "$novanodes"
             fi
+
+            if [ -n "$want_sles12" ] && [ -n "$want_docker" ] ; then
+                proposal_set_value nova default "['deployment']['nova']['elements']['nova-multi-compute-docker']" "['$sle12node']"
+                # do not assign another compute role to this node
+                local novanodes
+                novanodes=`printf "%s\n" $novanodes | grep -iv $sle12node`
+                novanodes="[ ${novanodes} ]"
```

Comment:
> Its not 'another'

I said "another" because it's the second one I encountered this morning ;-/

> We are working on refactoring mkcloud/qa_crowbarsetup with the goal to split it into smaller pieces (in order to transform the smaller pieces easier into other languages).

Yes, I think I was the one who suggested doing that ;-)

> This is tricky and takes time.

Of course.

> Until then, this is an exercise to learn to be patient :)

Sure, I'm waiting patiently :-)  But I also wanted to highlight a real-world example of shell causing issues, to make sure that people know there are valid reasons for my rants about shell ;-)